airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/ConfigFetchActivityImpl.java

```diff
@@ -30,31 +32,53 @@
 import java.util.Date;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
 import java.util.TimeZone;
 import java.util.UUID;
 import java.util.function.Supplier;
 import lombok.extern.slf4j.Slf4j;
 import org.joda.time.DateTimeZone;
 import org.quartz.CronExpression;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Slf4j
 @Singleton
 public class ConfigFetchActivityImpl implements ConfigFetchActivity {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigFetchActivityImpl.class);
   private final static long MS_PER_SECOND = 1000L;
   private final static long MIN_CRON_INTERVAL_SECONDS = 60;
+  private static final Set<UUID> SCHEDULING_NOISE_WORKSPACE_IDS = Set.of(
+      // Testing
```

Comment:
Add a TODO to move this to env vars?

Reply:
Hoping we'll get rid of it altogether :)